tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -535,7 +535,7 @@ def check_args(kwargs):
 
     if kwargs['extra_prefs']:
         # If a single pref is passed in as a string, make it a list
-        if type(kwargs['extra_prefs']) in (str, unicode):
+        if isinstance(kwargs['extra_prefs'], text_type):
```

Comment:
Hmm, the current code also allows `binary_type` on Py2. Are we sure this is correct?

Reply:
You are right. I need to check on binary as well. Code updated. Thank you!